### PR TITLE
Added sample, and preserve attribute

### PR DESCRIPTION
**Pull request by Clancey**
### Description of Change ###

Adds a Preserve attribute on the embedded font loaders.
Also adds a sample showing embedded fonts

### API Changes ###
none

### Platforms Affected ### 
- Core/XAML (all platforms)
- iOS
- Android

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj [hunk 1]
```diff
@@ -38,8 +42,27 @@
...
+      <Generator></Generator>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Fonts\PTM55FT.ttf">
+      <SubType></SubType>
+      <Generator></Generator>
+    </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Fonts\" />
```
Comment (PureWeen) [inline Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj]: I think we can remove this

File: Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj [hunk 2]
```diff
@@ -22,6 +22,10 @@
   </PropertyGroup>
   <ItemGroup>
     <None Remove="BuildNumber.txt" />
+    <None Remove="Fonts\CuteFont-Regular.ttf" />
+    <None Remove="Fonts\Dokdo-Regular.ttf" />
+    <None Remove="Fonts\fa-regular-400.ttf" />
+    <None Remove="Fonts\PTM55FT.ttf" />
```
Comment (PureWeen) [inline Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj]: I think we can remove this entire ItemGroup

Comment (Clancey) [reply Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj]: VS4mac added that. I tried removing it. Then they were duplicated.

Comment (PureWeen) [reply Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj]: Yea I'm trying it on vsmac as well and it's not reacting so well :-/ 

So we can leave for now

File: Xamarin.Forms.Platform.iOS/EmbeddedFontLoader.cs [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 namespace Xamarin.Forms.Platform.iOS
 {
+	[Preserve(AllMembers =true)]
```
Comment (PureWeen) [inline Xamarin.Forms.Platform.iOS/EmbeddedFontLoader.cs]:
```suggestion
	[Preserve(AllMembers = true)]
```

**Pull request merged**